### PR TITLE
Enable metrics-export-port by default and configure prometheus monitoring

### PR DESCRIPTION
**Pull request by scarlet25151**
@Jeffwan 

## Why are these changes needed?

This pr will enable the specific prometheus stack for kuberay, including some setting for service monitor and the configurations for the grafana dashboard.

## Checks

- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by Jeffwan** [comment]
image
image
image

**Comment by harryge00** [comment]
Great PR! I think we should also update docs on how to enable monitoring?

**Comment by Jeffwan** [comment]
image

@scarlet25151 Seems the unit test doesn't pass, please take a look and fix it.

**Comment by Jeffwan** [comment]
@scarlet25151 can you also address @harryge00's comment? let's add a short doc on the usage.

**Comment by scarlet25151** [comment]
Hi @harryge00 @Jeffwan I've address the comments and plz take a look.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,18 @@ kubectl apply -k "github.com/ray-project/kuberay/manifests/base?ref=v0.2.0"
 A helm chart is a collection of files that describe a related set of Kubernetes resources. It can help users to deploy ray-operator and ray clusters conveniently.
 Please read [kuberay-operator](helm-chart/kuberay-operator/README.md) to deploy an operator and [ray-cluster](helm-chart/ray-cluster/README.md) to deploy a custom cluster.
 
+### Monitor
+
+We have add a paramaeter `--metrics-expose-port=8080` to open the port and expose metrics both for the ray cluster and our control plane. We also leverage the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) to start the whole monitoring system.
```
Comment (harryge00) [inline README.md:43]: typo `parameter`

File: README.md [hunk 2]
```diff
@@ -38,6 +38,18 @@ kubectl apply -k "github.com/ray-project/kuberay/manifests/base?ref=v0.2.0"
...
+
+You can quickly deploy one by the following on your on kubernetes cluster by using the scripts in install:
```
Comment (harryge00) [inline README.md:45]: `on your own kubernetes cluster` ?

**Review by harryge00 — CHANGES_REQUESTED** [review]
Just two typos

**Comment by harryge00** [comment]
LGTM after fixing typos

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**